<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ROV_groundstation</class>
 <widget class="QWidget" name="ROV_groundstation">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1415</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ROV_groundstation</string>
  </property>
  <widget class="QGroupBox" name="ControlPanel">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>20</y>
     <width>561</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Position/Attitute Control Panel</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Deep Control</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="yValueScrollBar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>240</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-32768</number>
    </property>
    <property name="maximum">
     <number>32768</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QDial" name="yawValueDial">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-32768</number>
    </property>
    <property name="maximum">
     <number>32768</number>
    </property>
   </widget>
   <widget class="QScrollBar" name="xValueScrollBar">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>32</width>
      <height>240</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-32768</number>
    </property>
    <property name="maximum">
     <number>32768</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="zValueScrollBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>32</width>
      <height>240</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-32768</number>
    </property>
    <property name="maximum">
     <number>32768</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Yaw</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="lightsControl">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>410</y>
     <width>561</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Lights/Motor Control</string>
   </property>
   <widget class="QCheckBox" name="lightsStateCheckBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Lights On/Off</string>
    </property>
   </widget>
   <widget class="QSlider" name="lightBrightnessSlider">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>43</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Motor Unlock</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>721</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>ROV State Monitor Panel</string>
   </property>
   <widget class="QLabel" name="labelROVPings">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>491</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>PingState</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ROV Alive</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="CamMoniter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>721</width>
     <height>511</height>
    </rect>
   </property>
   <property name="title">
    <string>ROV Camera Moniter</string>
   </property>
   <widget class="QLabel" name="labelROVcam">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>36</y>
      <width>691</width>
      <height>431</height>
     </rect>
    </property>
    <property name="text">
     <string>video</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCamState">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>691</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CamState</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>580</y>
     <width>561</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>ROV Sensor Monitor</string>
   </property>
   <widget class="QSlider" name="depthSlider">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>16</width>
      <height>111</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-300</number>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="depthLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Depth</string>
    </property>
   </widget>
   <widget class="QLabel" name="rowLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Row</string>
    </property>
   </widget>
   <widget class="QLabel" name="yawLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Yaw</string>
    </property>
   </widget>
   <widget class="QLabel" name="pitchLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Pitch</string>
    </property>
   </widget>
   <widget class="QLabel" name="xaccLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X Acc</string>
    </property>
   </widget>
   <widget class="QLabel" name="yaccLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Acc</string>
    </property>
   </widget>
   <widget class="QLabel" name="zaccLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Z Acc</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>721</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Connection</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
